src/main/java/dan200/computercraft/core/apis/HTTPAPI.java

```diff
@@ -18,56 +18,45 @@
 
 import static dan200.computercraft.core.apis.ArgumentHelper.*;
 
-public class HTTPAPI implements ILuaAPI
-{
+public class HTTPAPI implements ILuaAPI {
     private final IAPIEnvironment m_apiEnvironment;
     private final List<HTTPTask> m_httpTasks;
-    
-    public HTTPAPI( IAPIEnvironment environment )
-    {
+
+    public HTTPAPI(IAPIEnvironment environment) {
         m_apiEnvironment = environment;
         m_httpTasks = new ArrayList<>();
     }
-    
+
     @Override
-    public String[] getNames()
-    {
-        return new String[] {
-            "http"
+    public String[] getNames() {
+        return new String[]{
+                "http"
         };
     }
 
     @Override
-    public void startup( )
-    {
+    public void startup() {
     }
 
     @Override
-    public void advance( double _dt )
-    {
+    public void advance(double _dt) {
         // Wait for all of our http requests 
-        synchronized( m_httpTasks )
-        {
+        synchronized (m_httpTasks) {
             Iterator<HTTPTask> it = m_httpTasks.iterator();
-            while( it.hasNext() )
-            {
+            while (it.hasNext()) {
                 final HTTPTask h = it.next();
-                if( h.isFinished() )
-                {
-                    h.whenFinished( m_apiEnvironment );
+                if (h.isFinished()) {
+                    h.whenFinished(m_apiEnvironment);
                     it.remove();
                 }
             }
         }
     }
-    
+
     @Override
-    public void shutdown( )
```

Comment:
I don't like to nitpick, but it's probably best to keep whitespace changes to a minimum. I know indent styles are rather a personal matter, but it's best to be consistent :). It also makes it easier to review the PR and see what functionality has changed, rather than what's just cosmetic.

Reply:
BTW, it really annoys me that the {'s seem to be on a separate line than the code controlling  the block, but I try to follow that formatting anyways. Don't know why.

Reply:
Yeah, there's lots of things I'm not a fan of as far as CC's formatting goes, but you have to roll with it :).